src/components/common/customizableAside.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import CustomizableAvatar from "./customavatar/CustomizableAvatar";
+
+function CustomizableAside({
```

Comment:
@TaylorDJones11 Did you change all the imports of `CustomizableAside` to this new location?

Reply:
Yes I did